src/Symfony/Component/VarDumper/composer.json

```diff
@@ -26,7 +26,9 @@
         "phpunit/phpunit": "<4.8.35|<5.4.3,>=5.0"
     },
     "suggest": {
-        "ext-symfony_debug": ""
+        "ext-iconv": "Required to convert non-UTF-8 strings to UTF-8.",
+        "ext-symfony_debug": "",
+        "symfony/polyfill-iconv": "In case ext-iconv cannot be used."
```

Comment:
I would not add this one. The suggestion would be displayed to everyone having iconv, which will make suggestions harder to read.

I would rather mention it in the ext-iconv suggestion, in case they cannot get the extension